website/src/types/Chat.ts

```diff
@@ -184,4 +184,5 @@ export interface GetChatsParams {
   limit?: number;
   before?: string;
   after?: string;
+  include_hidden?: string;
```

Comment:
I think we should have an option to only show hidden chat, since include_hidden will result in all chats, we still have to make the UI to show whether the chat is hidden or not.

Reply:
true, this will require some changes in the backend, so I am going to move it to a different PR.

Reply:
If you are going to work in the inference backend, I suggest we have a filter like this, and apply when it is not null
```ts
type Filter = {
  hidden?: boolean
  allow_data_use?: boolean
  shared?: boolean // future
}
```

So the user can compose the filter they want. I can work on the frontend UI